packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -30,8 +30,10 @@
 		font-family: $default-font;
 
 		// Increase specificity.
-		.wp-block-navigation-item {
+		.wp-block-navigation-item,
+		.wp-block-navigation-submenu {
```

Comment:
I don't think we need this anymore because submenu blocks have both classnames as of #33918.

Reply:
Cool, done 👍